packages/editor/src/components/post-title/index.native.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { TextInput } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { decodeEntities } from '@wordpress/html-entities';
+import { ENTER } from '@wordpress/keycodes';
+import { withInstanceId, compose } from '@wordpress/compose';
+
+class PostTitle extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.onChange = this.onChange.bind( this );
+		this.onSelect = this.onSelect.bind( this );
+		this.onUnselect = this.onUnselect.bind( this );
+		this.onKeyDown = this.onKeyDown.bind( this );
+		this.redirectHistory = this.redirectHistory.bind( this );
+
+		this.state = {
+			isSelected: false,
+		};
+	}
+
+	handleFocusOutside() {
+		this.onUnselect();
+	}
+
+	onSelect() {
+		this.setState( { isSelected: true } );
+		this.props.clearSelectedBlock();
+	}
+
+	onUnselect() {
```

Comment:
handleFocusOutside, onSelect and onUnselect functions do not seemed to be called from anywhere, planning to bind these in the future?

Reply:
Just pushed a commit where `onSelect` is wired.

Regarding `onUnselect` it's called from `handleFocusOutside`, which I believe is called by some other editor component - in fact if you check the original (web) .js sources, `handleFocusOutside` is never called.